system-test/busybench/package.json

```diff
@@ -20,5 +23,7 @@
     "posttest": "npm run check"
   },
   "devDependencies": {},
-  "dependencies": {}
+  "dependencies": {
+    "@google-cloud/profiler": "^4.2.0"
```

Comment:
I think this change means the system test will always run with @google-cloud/profiler 4.2.0?

If so, that would mean the system test won't work anymore